packages/styles/header.import.styl

```diff
@@ -99,6 +110,62 @@ header
       margin 15px auto 8px auto
       padding 15px
       text-align center
+    .open
+      .dropdown-menu
+        border-top 4px solid $primary-dark
+          > li
+            border-bottom 1px solid $secondary-light
+            &:last-of-type
+              border 0
+            > a
+              color white
+              padding 10px 15px 10px 45px
+              font-size 12px
+              &:hover
+                background $primary
+    > li > a.navbar-btn
+      width 50%
+      text-align center
+      margin 15px auto 8px auto
+      padding 15px
+  .dropdown-menu
+    padding-top 0
+    border-top 2px solid $primary
+    > li
+      border-bottom 1px solid $border-gray
+      text-shadow none
+      &:last-of-type
+        border 0
+      > a
+        clearFix()
+        position relative
+        padding 8px 20px
+        text-shadow none
+        +below(1)
+          padding 8px 35px !important
```

Comment:
I think I structured the stylus haphazardly when implementing rupture. I did a good bit of refactoring to hopefully clear things up. The styles are still pretty complex but its mostly because of bootstrap's nested selections.